tumblr-gif-search/app.py

```diff
@@ -98,27 +93,17 @@ def main(task, num_docs_query, num_docs_index):
             sys.exit(1)
     print(f'### task = {task}')
```

Comment:
remove prints

Reply:
Removed in https://github.com/jina-ai/examples/pull/460/commits/8f96d15783f19b7d22e3294bcdf8c961c61c7723